.github/workflows/main.yml

```diff
@@ -0,0 +1,63 @@
+name: Run tests
+
+on:
+  [push, pull_request]
```

Comment:
@vj-codes , how come the 'develop' branch is not specify here? atm this PR is not triggering a gh-action build. Perhaps this has something to do with it?

Reply:
the current code will trigger gh actions every time a contributor pushes the changes on their fork as well as every time a pull request is created here irrespective of any branch .
The build is passing under actions on my fork and will be triggered by other PRs here after this is merged.

Reply:
@vj-codes , I believe if we speccify the `develop` as the target branch for pull and push, it will trigger the gh-action here on this PR even before you merge it. This way we can see if the build pass or not before merging the code, which is much better than if we wait till the PR is merged and wait for another PR to be pushed to check if it works or not. You can see this example on my MS PR [# 1029](https://github.com/anitab-org/mentorship-backend/pull/1029). Initially I left the branch as develop as that's what the existing code has (as `bit` branch was branched out of the `develop` branch). However, since it didn't trigger the build, I changed it to `bit`. Only then it triggered the build there. 
IMO for this PR, it is more important to make sure that the build is passing on the actual `develop` branch before the code being merge rather than to merge first then triggers the build later with the next PR. What do you think?